gallery/how_to/work_with_microtvm/micro_custom_ide.py

```diff
@@ -0,0 +1,361 @@
+        "https://github.com/tlc-pack/web-data/raw/main/testdata/microTVM/data/vww_sample_person.jpg"
+    )
+    SAMPLE_NOT_PERSON_URL = "https://github.com/tlc-pack/web-data/raw/main/testdata/microTVM/data/vww_sample_not_person.jpg"
+
+    SAMPLE_PERSON_PATH = download_testdata(SAMPLE_PERSON_URL, "person.jpg", module=SAMPLES_DIR)
+    img = Image.open(SAMPLE_PERSON_PATH)
+    create_header_file("sample_person", np.asarray(img), SAMPLES_DIR, tar_file)
+
+    SAMPLE_NOT_PERSON_PATH = download_testdata(
+        SAMPLE_NOT_PERSON_URL, "not_person.jpg", module=SAMPLES_DIR
+    )
+    img = Image.open(SAMPLE_NOT_PERSON_PATH)
+    create_header_file("sample_not_person", np.asarray(img), SAMPLES_DIR, tar_file)
+
+######################################################################
+# At this point you have all you need to take the compiled model to your IDE and evaluate it. Inside the MLF file (model.tar), you should find the following file hirearchy:
+#
+#     .. code-block::
+#
+#      /root
+#      ├── codegen
+#      ├── parameters
+#      ├── runtime
+#      ├── samples
+#      ├── src
+#      ├── templates
+#      ├── metadata.json
+#
+# * The codegen folder includes the C code TVM generated for your model.
+# * The runtime folder includes all the TVM dependencies that the target needs to compile the generated C code.
+# * The samples folder includes the two generated sample files for evauating the model.
+# * The src folder include the relay module describing the model.
+# * The templates folder include two template files that you might need to edit based on your platform.
+# * The metadata.json file includes information about the model, its layers and memory requirement.
+#
+
+
+######################################################################
+# Generate the project in your IDE
+# -----------------------------------
+#
+# The next step is to create a project for our target device. We use STM32CubeIDE, you can download it `here <https://www.st.com/en/development-tools/stm32cubeide.html>`_.
+# We are using version 1.11.0 in this tutorial. Once you install STM32CubeIDE follow these steps to create a project:
+#
+# #. select File -> New -> STM32Project. The target selection Window appears.
+#
+# #. Navigate to the "Board Selector" tab, type in the board name "nucleo-l4r5zi" in the "Commercial Part Number" text box. Select the board from the list of boards that appear on the right side of the screen and click "Next".
+#
+# #. Type in your project name (for example microtvm_vww_demo). We are using the default options. (Target Language: C, Binary Type: Executable, Project Type: STM32Cube). Click "Finish".
+#
+# #. A text box will appear asksing if you want to "Initialize all the peripherals with their default mode?". click "Yes". This would generate the project and open the device configuration tool where you can use the GUI to setup the peripherals. By default the USB, USART3 and LPUART1 are enabled, as well as a few GPIOs.
```

Comment:
```suggestion
# #. A text box will appear asking if you want to "Initialize all the peripherals with their default mode?". click "Yes". This would generate the project and open the device configuration tool where you can use the GUI to setup the peripherals. By default the USB, USART3 and LPUART1 are enabled, as well as a few GPIOs.
```